protected/models/forms/project/SyncWorkspaces.php

```diff
@@ -0,0 +1,64 @@
+<?php
+declare(strict_types=1);
+
+namespace prime\models\forms\project;
+
+use Carbon\Carbon;
+use prime\models\ar\Project;
+use prime\models\ar\Workspace;
+use prime\objects\BatchResult;
+use yii\base\Model;
+use yii\helpers\Html;
+use yii\validators\RangeValidator;
+use yii\validators\RequiredValidator;
+use function iter\keys;
+use function iter\toArray;
+
+class SyncWorkspaces extends Model
+{
+
+    private Project $project;
+
+    public array $workspaces = [];
+
+
+    public function __construct(
+        Project $project,
+        $config = []
+    ) {
+        parent::__construct($config);
+        $this->project = $project;
+    }
+
+    public function workspaceOptions(): iterable
```

Comment:
Should the form model know that it being rendered to HTML? Why not return an array with the keys containing the id and the value just an array which you can combine in the view to the desired layout?